src/System.Security.Cryptography.X509Certificates/tests/CertTests.cs

```diff
@@ -187,6 +191,29 @@ public static void X509Cert2ToStringVerbose()
             }
         }
 
+        [Theory]
+        [MemberData(nameof(StorageFlags))]
+        public static void X509Certificate2ToStringVerbose_WithPrivateKey(X509KeyStorageFlags keyStorageFlags)
+        {
+            using (var cert = new X509Certificate2(TestData.PfxData, TestData.PfxDataPassword, keyStorageFlags))
+            {
+                string certToString = cert.ToString(true);
```

Comment:
Nit: it'd be helpful for the bool arg to be named at the call site (here and in the test below).  No need to restart CI for it though.